### PR TITLE
bootstrap javascript package

### PR DESCRIPTION
**Pull request by gberenfield**
Package for the js for bootstrap components

### CODE REVIEW

File: bootstrap/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# cljsjs/bootstrap
+[](dependency)
+```clojure
+[cljsjs/jquery "1.9.1-0"] ;; latest release
```
Comment (ducky427) [inline bootstrap/README.md]: This should probably be `[cljsjs/bootstrap "3.3.5-0"]`

**Comment by gberenfield** [comment]
Sorry, fixed in my local branch. Should I close this pull request and create a new one?

**Comment by ducky427** [comment]
I don't think you need to close the PR. The existing PR has gotten updated with your latest changes.

**Comment by gberenfield** [comment]
OK thanks

**Comment by martinklepsch** [comment]
Thanks. Published to clojars :+1:

**Pull request closed without merging**